prototypes/particles.lua

```diff
@@ -29,7 +29,12 @@ napalm_particle.tint = napalm_tint
 
 local napalm_fire_flame = table.deepcopy(data.raw["fire"]["fire-flame"])
 napalm_fire_flame.name = "napalm_fire_flame"
-napalm_fire_flame.emissions_per_second = 0.050
+--napalm_fire_flame.emissions_per_second = 0.050
+--change in 2.0 modding API
```

Comment:
We should remove the "added due to 2.0 modding API changes" comments, as the git commit has indicated the changes.